tests/StubsPhpDocTest.php

```diff
@@ -126,7 +126,8 @@ private static function checkHtmlTags(BasePHPElement $element, string $elementNa
                 '#`.*<.*>`#U',
                 '#<pre>.*</pre>#sU',
                 '#<code>.*</code>#sU',
-                '#@author.*<.*>#U'
+                '#@author.*<.*>#U',
+                '#array<.*>#sU'
```

Comment:
Ah i see the problem, it is because some doc comments also contain something like:
```
<b>pg_fetch_array</b>
```
which will match and just leave
```
<b>pg_fetch_
```
so i could go with `'# array<.*>#sU', but i am not sure if that is the nicest way, since probably it should be possible to have a whitespace or a pipe before the pattern. I am not so versed with patterns, maybe somebody knows how to best solve this :sweat_smile:

Reply:
This will help you: https://regex101.com/

Reply:
Thanks a lot, that page is a lifesaver :tada: